README.md

```diff
@@ -88,93 +76,107 @@ Destroy the stream. Closes all feeds as well.
 
 Gracefully end the stream. Closes all feeds as well.
 
-#### `feed.info(message)`
+#### `feed.options(message)`
 
-Send an `info` message. See the [schema.proto](schema.proto) file for more information.
+Send an `options` message.
```

Comment:
Might be a PITA but a link to the various messages' schemas would be useful in these different calls.

Reply:
done